GPCS4/Graphic/Gnm/GnmCommandBufferDraw.cpp

```diff
@@ -415,7 +415,16 @@ void GnmCommandBufferDraw::drawIndexAuto(uint32_t indexCount)
 
 void GnmCommandBufferDraw::setEmbeddedVsShader(EmbeddedVsShader shaderId, uint32_t shaderModifier)
 {
-	
+	const uint8_t embeddedVsShaderFullScreen[] = 
```

Comment:
function scope should be enough.
https://github.com/Inori/GPCS4/blame/master/Doc/CodeStyle.md#L202